src/services/m3u-service.js

```diff
@@ -0,0 +1,149 @@
+const CacheService = require('../cache-service');
+const constants = require('../constants');
+
+/**
+ * Manager and Generate M3U content
+ *
+ * @class M3uService
+ */
+class M3uService {
```

Comment:
Small nitpick: We name this `M3uService`, but we have a method named `buildM3UList`. Could we standardize on either `M3u` or `M3U`?